lib/BC/SleighLifter.cpp

```diff
@@ -1263,6 +1267,27 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock {
                                                   vars[2]);
         return kLiftedInstruction;
       }
+      if (other_func_name == kSysCallName &&
+          insn.arch_name == ArchName::kArchPPC) {
+        DLOG(INFO) << "Invoking syscall";
+
+        const auto [mem_ptr_ref, mem_ptr_ref_type] =
+            insn_lifter_parent.LoadRegAddress(
```

Comment:
Is there a reason `remill::LoadMemoryPointerRef` doesnt work here?

Reply:
[5de355d](https://github.com/lifting-bits/remill/pull/653/commits/5de355d1253bd84460bad7cbb60e178c21c65421)